lib/bunny/session.rb

```diff
@@ -303,10 +307,6 @@ def start
           @transport.post_initialize_socket
           @transport.connect
 
-          if @socket_configurator
```

Comment:
Why were these lines removed?

Reply:
I couldn't find any usage of that instance variable. How would you set it? `session.set_instance_variable('@socket_configurator', ...)` ?
Besides it looks, as if you can just use the `configure_socket` method of the `Session` class to reach the same thing. So in my opinion these lines were useless or do you have a usecase?